components/console/helpers/table.rst

```diff
@@ -150,6 +150,30 @@ The output of this command will be:
     |                (the rest of the rows...)            |
     +-------+------------+--------------------------------+
 
+By default, the table is rendered horizontally, which may not look great if
+the number of columns you want to display or the data they contain is too large.
+
+If you prefer to render your table vertically, use the
```

Comment:
I would instead write:

« Console tables content are displayed horizontally by default.
You can change this behavior via the setVertical() method » (and the example: for me the example is more worth it that the full description)

also I think you may need to add a version added directive as of 6.1 :)